DontPanic Code of Conduct.md

```diff
@@ -0,0 +1,39 @@
+{{TOC}}
+
+# DontPanic Code of Conduct
+
+## Our Pledge
+
+Our goal as the contributors, developers, and maintainers of DontPanic is to create an environment that is inclusive, caring and non-discriminatory.  We are fully accepting and and welcoming of all people regardless of their identity preferences or backgrounds.
+
+We do so to facilitate a healthy and inclusive community so that productivity can be emphasized along side a healthy work environment.
+
+## Standards for Behavior
+Guidelines for acceptable behavior include but are not limited to:  
+* Using welcoming language  
+* Respecting other contributors and their viewpoints/beliefs  
+* Focusing on the bigger picture. Development should not be about the self, but about the community.  
+* Being empathetic towards others.  
+* The peaceful and mature resolution of any conflicts that arise, and if necessary, with the help of a mediator.  
+
+Behaviors that are not acceptable include:  
+* Harassment of any sort including verbal, physical, and sexual.  
+* Dishonesty, withholding of vital information, lying or manipulation.  
+* Personal attacks of any kind based on a persons characteristics.  
+* Criticism that is not intended to be constructive, but hurtful.  
+
+## Responsibilities
+The leaders of the DontPanic project are able to clarify and enforce the standards set forth, and will use their judgement to decide if any behavior not explicitly defined in the Standards for Behavior is unethical enough to warrant removal from contributing DontPanic.
+The leaders of DontPanic also have final authority with regards to all forms of editing or removing code from the main repository.
+
+## Scope  
+This Code of Conduct has global scope, and applies to all areas, virtual or real, where any form of DontPanic is being discussed or developed.
+
+## Enforcement
+Violations of the Standards for Behavior, any form or harassment, or any other behavior that warrants investigation can be reported to <ras472@nau.edu>.
+Complaints will be examined and a response will be delivered in a timely manner that may or may not include actions that will be taken.
+
+## Attribution  
+This Code of Conduct was based upon the template provided by [Contributor Covenant][homepage],
```

Comment:
```suggestion
This Code of Conduct was based upon the template provided by Contributor Covenant,
```

This does not create a hyperlink. Easiest fix is just to remove the brackets, as I have done here. If you'd rather reformat as a hyperlink, feel free to do that instead.